packages/litqa/README.md

```diff
@@ -0,0 +1,19 @@
+# aviary.litqa
+
+LitQA environment implemented with aviary,
+allowing agents to perform question answering on the LitQA dataset.
+
```

Comment:
Can you document the difference between LitQA and LitQA2 (it's described in LAB-Bench paper  https://arxiv.org/abs/2407.10362)